src/app.js

```diff
@@ -64,12 +64,14 @@ export default function createApp(rootElement) {
   });
   rootElement.appendChild(listHeader);
 
-  const todoList = createTodoList();
+  const todoList = createTodoList("todo-list");
   rootElement.appendChild(todoList);
 
   let todoItems = [];
   let userName;
 
+  // TODO: extract all local storage related functions
+  // TODO: pass in username as parameter
```

Comment:
next PR should focus on extracting the local storage related code from `createApp`